variables.tf

```diff
@@ -55,3 +55,9 @@ variable "actions_ok" {
   default     = []
   description = "A list of actions to take when alarms are cleared. Will likely be an SNS topic for event distribution."
 }
+
+variable "linear_fill_missing_data" {
```

Comment:
can we make the metrics `expression` configurable instead?

Reply:
I can look into that.  The one downside to that approach would be that we'd need to specific a default value for that `expression` as a variable to keep this backwards compatible.

Maybe that could work without scarifying clarity.  I'll see.

Reply:
You can do `""` and then check `length(var.expression) > 0`

Reply:
@tovbinm Yeah, I think that is a little cleaner. Check out the latest commit.

Reply:
yep!